frontend/src/pages/agreements/StepCreateAgreement.jsx

```diff
@@ -1,31 +1,30 @@
 import React from "react";
-import PropTypes from "prop-types";
 import { useNavigate } from "react-router-dom";
 import { useDispatch, useSelector } from "react-redux";
 import StepIndicator from "../../components/UI/StepIndicator/StepIndicator";
-import ProcurementShopSelect from "./ProcurementShopSelect";
-import AgreementReasonSelect from "./AgreementReasonSelect";
-import AgreementTypeSelect from "./AgreementTypeSelect";
-import ProductServiceCodeSelect from "./ProductServiceCodeSelect";
+import ProcurementShopSelect from "../../components/Agreements/ProcurementShopSelect/ProcurementShopSelect";
+import AgreementReasonSelect from "../../components/Agreements/AgreementReasonSelect/AgreementReasonSelect";
+import AgreementTypeSelect from "../../components/Agreements/AgreementTypeSelect/AgreementTypeSelect";
+import ProductServiceCodeSelect from "../../components/Agreements/ProductServiceCodeSelect/ProductServiceCodeSelect";
 import Alert from "../../components/UI/Alert/Alert";
 import {
     setAgreementDescription,
     setAgreementId,
     setAgreementIncumbent,
     setAgreementNotes,
-    setAgreementProjectOfficer,
-    setAgreementTeamMembers,
+    setAgreementProcurementShop,
     setAgreementTitle,
-    setSelectedAgreementReason,
-    setSelectedProcurementShop,
+    setSelectedProcurementShop as setSelectedProcurementShopInAgreement,
     setSelectedProject,
 } from "./createAgreementSlice";
-import ProjectOfficerSelect from "./ProjectOfficerSelect";
-import TeamMemberSelect from "./TeamMemberSelect";
-import TeamMemberList from "./TeamMemberList";
+import { setSelectedProcurementShop as setSelectedProcurementShopInBudgetLine } from "../budgetLines/createBudgetLineSlice";
+import ProjectOfficerSelect from "../../components/Agreements/ProjectOfficerSelect/ProjectOfficerSelect";
```

Comment:
could we add an `index.js` to the Component folder adding a default export in order to not double the folder/component?